packages/paypal-js/e2e-tests/load-cached-script.html

```diff
@@ -13,6 +13,7 @@ <h1>Load Cached Script</h1>
             const options = {
                 clientId: "test",
                 dataPageType: "checkout",
+                dataJsSdkLibrary: "paypal-js",
```

Comment:
This smells funky, because `dataJsSdkLibrary` is not a required script option, but if we don't add it here, the "load cached script" tests fail.